__tests__/patch.js

```diff
@@ -118,6 +150,30 @@ describe("delete 1", () => {
     )
 })
 
+describe("delete 2", () => {
+    runPatchTest(
+        new Map([["x", 1]]),
+        d => {
+            d.delete("x")
+        },
+        [{op: "remove", path: ["x"]}],
```

Comment:
Are `Map` entries assumed to be JSON-compatible?

Reply:
I guess one could serialize a Map to an object if needed. So not, not out-of-the-box. but yes with certain additional work.